src/Kunstmaan/NodeSearchBundle/Helper/ElasticSearchUtil.php

```diff
@@ -12,6 +14,16 @@
      */
     public static function useVersion6()
     {
-        return (PHP_MAJOR_VERSION == 7 && !class_exists('\Elastica\Tool\CrossIndex'));
+        $client = ClientBuilder::create()->build();
+        $info = $client->info();
+
+        if (null !== $info) {
+            $versionParts = explode('.', $info['version']['number']);
+            $majorVersion = $versionParts[0];
+
+            return ($majorVersion > 2 && PHP_MAJOR_VERSION === 7 && !class_exists('\Elastica\Tool\CrossIndex'));
```

Comment:
I propose to use $majorVersion >= 6

Reply:
@treeleaf Are the bundles changes working on elasticsearch 5 to ?

Reply:
It's also available in ES5, so why checking only for >= 6 ?

Reply:
I tested it also with ES5. 
Maybe as a second thought, we should first check on `!class_exists('\Elastica\Tool\CrossIndex')` and PHP version before fetching the ES version info. If this class exists, then we know version 5 of the library is installed. Also version 6 of the library only works in PHP 7.

Reply:
True, I changed the check priority.